api-docs/apiLive.htm

```diff
@@ -12803,7 +12803,7 @@ <h2 class="flybar-button">Template</h2>
 							numberOfRepaymentVariationsForBorrowerCycle,
```

Comment:
Add a field description : Ex:

allowAttributeOverrides : Specifies if select attributes may be overridden for individual loan accounts